test/reusing-handlers.test.ts

```diff
@@ -55,9 +55,6 @@ it('returns the mocked response from the middleware', async () => {
       expect.stringMatching(
         new RegExp(`GET ${httpServer.http.makeUrl('/user')}`),
       ),
-      expect.stringMatching(
```

Comment:
I'm not sure why, but it seems like a warning about `OPTIONS` request doesn't exist.